applications/Unity.GrantManager/src/Unity.GrantManager.Domain/Assessments/AssessmentManager.cs

```diff
@@ -58,20 +54,25 @@ public async Task<Assessment> CreateAsync(
 
         var form = await _applicationFormRepository.GetAsync(application.ApplicationFormId);
 
+        var otherAssessments = await _assessmentRepository.GetListByApplicationId(application.Id);
+        bool hasOtherAssessments = otherAssessments != null && otherAssessments.Count != 0;
+
         var assessment = await _assessmentRepository.InsertAsync(
             new Assessment(
                 GuidGenerator.Create(),
                 application.Id,
                 assessorUser.Id),
             autoSave: true);
+        
 
-        if (form.ScoresheetId != null && await _featureChecker.IsEnabledAsync("Unity.Flex"))

```

Comment:
should we not leave this feature check in place?

Reply:
hi @AndreGAot -- i removed the feature checking so that the scoresheet can still be created on the assessment record even if the user that clicked the "Create Assessment" button does not have the Unity.Flex feature

Reply:
> should we not leave this feature check in place?

@AndreGAot - Hi Andre, done restoring back the feature check. thanks.